ci/Dockerfile

```diff
@@ -14,10 +14,11 @@
 # limitations under the License.
 #
 
-ARG CUDA_VERSION=11.8.0
-FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu20.04
+ARG CUDA_VERSION=12.1.0
```

Comment:
Q: why 12.1 here?

Reply:
Check the tags of nvidia/cuda, will alternative to [11.8.0-devel-ubuntu22.04](https://hub.docker.com/layers/nvidia/cuda/11.8.0-devel-ubuntu22.04/images/sha256-60eda04ab6790aa76d73bf0df245b361eabc6d8f7b6f6cf9846c70f399b9a1eb)